packages/@aws-cdk/aws-ecs-patterns/lib/fargate/network-load-balanced-fargate-service.ts

```diff
@@ -135,5 +140,11 @@ export class NetworkLoadBalancedFargateService extends NetworkLoadBalancedServic
       cloudMapOptions: props.cloudMapOptions,
     });
     this.addServiceAsTarget(this.service);
+
+    // all public-facing NLB fronted fargate service with assignPublicIp should allow from any IPv4 traffic
+    const publicNLB = props.publicLoadBalancer !== undefined ? props.publicLoadBalancer : true;
+    if (publicNLB && props.assignPublicIp) {
+      this.service.connections.connections.allowFromAnyIpv4(Port.tcp(containerPort));
+    }
```

Comment:
Please submit this change as a separate PR.

Reply:
OK removed.